### PR TITLE
suggest that the key might have been expired if no valid primary key self-signature is available

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: src/openpgp/processor.iced [hunk 1]
```diff
@@ -60,7 +60,7 @@ class KeyBlock
 
   _check_primary : () ->
     err = if not @primary.is_self_signed()
-      new Error "no valid primary key self-signature"
+      new Error "no valid primary key self-signature or signature(s) have expired"
```
Comment (maxtaco) [inline src/openpgp/processor.iced]: well it's not that signature have expired --- it's that the key producing the signatures have expired.

**Comment by maxtaco** [comment]
also, you have to build the .js file via `make`

**Comment by songgao** [comment]
Ah, I see! Sorry

**Comment by songgao** [comment]
Do we wanna bump version in `package.json` as well or that should go to a separate commit?

**Comment by maxtaco** [comment]
I'll do that as part of the release, thanks!

**Comment by maxtaco** [comment]
Sigh, I should have guessed that it would have force a rebuild of everything!  Shoot.  I meant to fix that about ICS.  Hmrg!

**Comment by maxtaco** [comment]
👍

**Pull request merged**